webui/react/src/types.ts

```diff
@@ -769,14 +771,57 @@ export const ExperimentItem = t.intersection([
     userId: t.number,
   }),
 ]);
-export type ExperimentItem = t.TypeOf<typeof ExperimentItem>;
+export type FullExperimentItem = t.TypeOf<typeof FullExperimentItem>;
+
+// difference from FullExperimentItem is that BulkExperimentItem doesnt have config and configRaw
+// bulk endpoints like experimentSearch dont return config due to perf issue
+// since https://github.com/determined-ai/determined/pull/8732
+export const BulkExperimentItem = t.intersection([
```

Comment:
only difference from `FullExperimentItem` is that `BulkExperimentItem` doesnt have `config` and `configRow`. `io-ts` doesnt support `omit` operator, so just list fields like this